src/shared/Core/CredentialCacheStore.cs

```diff
@@ -42,9 +45,10 @@ public IList<string> GetAccounts(string service)
             return Array.Empty<string>();
         }
 
+
         public ICredential Get(string service, string account)
         {
-            var input = MakeGitCredentialsEntry(service, account);
+            var input = MakeGitCredentialsEntry(service, new GitCredential(account, null));
```

Comment:
Can/should we add a `GitCredential` constructor that just takes an `account`?

Reply:
Done.